src/repository/screens/repository.screen.js

```diff
@@ -145,14 +161,16 @@ class Repository extends Component {
     const openIssues = pureIssues.filter(issue => issue.state === 'open');
     const closedIssues = pureIssues.filter(issue => issue.state === 'closed');
 
-    const repositoryActions = [starred ? '★ Unstar' : '★ Star'];
+    const repositoryActions = [starred ? 'Unstar' : 'Star'];
     const showFork =
       repository && repository.owner && repository.owner.login !== username;
 
     if (showFork) {
       repositoryActions.push('Fork');
     }
 
+    repositoryActions.push('Share');
```

Comment:
Could this all be simplified to:

```javascript
const showFork = repository && repository.owner && repository.owner.login !== username;
const repositoryActions = [starred ? 'Unstar' : 'Star', showFork && 'Fork', 'Share'];
```

Have no idea if that would work but might be worth trying it because it's a little hard to read as-is.